src/DisplayPlug.vala

```diff
@@ -208,6 +240,19 @@ public class Display.Plug : Switchboard.Plug {
         search_results.set ("%s → %s".printf (display_name, _("Screen mirroring")), "");
         return search_results;
     }
+
+    private void detect_touchscreen () {
+        var display = Gdk.Display.get_default ();
+        if (display != null) {
+            var manager = Gdk.Display.get_default ().get_device_manager ();
```

Comment:
There is no need to call `Gdk.Display.get_default ()` here again.
You can get the manager by just doing:
`var manager = display.get_device_manager ();`